client/post-editor/editor-sharing/publicize-connection.jsx

```diff
@@ -163,6 +167,35 @@ export class EditorSharingPublicizeConnection extends React.Component {
 		);
 	};
 
+	/**
+	 * If a connection needs reauthentication, display a warning linked to Sharing settings page so users can disconnect and reconnect.
+	 *
+	 * @returns {object} Warning about connection.
+	 */
+	renderMustReauthConnection = () => {
+		const { connection, siteSlug } = this.props;
+		if ( ! connection || connection.status !== 'must_reauth' ) {
+			return;
+		}
+
+		return (
+			<Notice
+				isCompact
+				className="editor-sharing__must-reauth-publicize-connection"
+				status="is-warning"
+				showDismiss={ false }
+			>
+				{ this.props.translate(
+					'Your LinkedIn connection needs to be reauthenticated to continue working – head to Sharing to take care of it.'
+				) }
+				<NoticeAction href={ addSiteFragment( '/sharing', siteSlug ) }>
+					{ this.props.translate( 'Go to Sharing settings' ) }{' '}
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'Go to Sharing Settings' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).